city_scrapers/spiders/alle_port_authority.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+import unicodedata
+from datetime import datetime, timedelta
+
+from dateutil.parser import parse
+from lxml import html
+
+from city_scrapers.constants import BOARD, COMMITTEE
+from city_scrapers.spider import Spider
+
+
+class AllePortAuthoritySpider(Spider):
+    name = 'alle_port_authority'
+    agency_name = 'Port Authority of Allegheny County'
+    timezone = 'America/New_York'
+    allowed_domains = ['www.portauthority.org']
+    start_urls = ['https://www.portauthority.org/paac/CompanyInfoProjects/'
+                  'BoardofDirectors/MeetingAgendasResolutions.aspx']
+    event_year = str(datetime.now().year)
+
+    def _build_datatable(self, response):
+        alist_tbody = (response
+                       .xpath('//table[1]/tbody//td')
+                       .extract()
+                       )
+
+        atable = []
+        arow = []
+
+        for item in alist_tbody:
+            tree = html.fragment_fromstring(item)
+            # pdb.set_trace()
+            text = tree.text_content()
+
+            url = tree.xpath('//a/@href')
+            find_att_b = tree.xpath('//b/text()|//strong/text()')
+            if len(find_att_b) >= 1:
+                continue
+            if url:
+                arow.append('{text}: {url}'
+                            .format(text=text, url=url[0]))
+            else:
+                arow.append('{text}'.format(text=unicodedata.normalize("NFKD", text)))
+            if len(arow) == 6:
+                atable.append(arow)
+                arow = []
+        return atable
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` a dict that follows a modified
+        OCD event schema (docs/_docs/05-development.md#event-schema)
+
+        Change the `_parse_id`, `_parse_name`, etc methods to fit your scraping
+        needs.
+        """
+        atable = self._build_datatable(response)
+        for row in atable:
+            data = {
+                'timezone': self.timezone,
```

Comment:
`timezone` isn't necessary to include here. Let me know if you see anything that says otherwise in the docs and I can update that